certs/rds/rds-us-west-2-bundle.pem

```diff
@@ -0,0 +1,76 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Just embed this file. Put it in internal/dsn/files or something and embed it as `[]byte`